src/utils.js

```diff
@@ -66,6 +66,41 @@ export function getDefaultRegistry() {
   };
 }
 
+const currentFormInstances = {};
+
+export function registerFormNode(formId, formNode) {
+  if (typeof formId !== "undefined") {
+    currentFormInstances[formId] = formNode;
+  }
+}
+
+export function unregisterFormNode(formId) {
+  delete currentFormInstances[formId];
+}
+
+function getRegisteredFormNode(formId) {
+  const formNode = currentFormInstances[formId];
+
+  if (!formNode && process.env.NODE_ENV === "development") {
+    console.error(
+      `No registered form node found with given id (${formId}). Please check if the target form has proper id assigned.`
+    );
+  }
+
+  return formNode;
+}
+
+export function submitForm(formId) {
+  const formNode = getRegisteredFormNode(formId);
+
+  if (formNode) {
+    formNode.dispatchEvent(new window.Event("submit", { cancelable: true }));
```

Comment:
I have to explicitly use `window.Event` here because of a test environment error (it seems related to test infra setup). If a reviewer can teach me how to fix below test environment issue. I'll update this PR by removing the `window`.

<img width="482" alt="Screen Shot 2019-08-17 at 12 28 24 AM" src="https://user-images.githubusercontent.com/6490166/63208393-3fcd1180-c088-11e9-8c42-61535de98e63.png">